app/views/articles/_single_story.html.erb

```diff
@@ -1,65 +1,65 @@
-  <div class="single-article single-article-small-pic">
-    <% if story.video.present? && story.video_thumbnail_url.present? %>
-      <a class="single-article-video-preview" style="background-image:url(<%= story.cloudinary_video_url %>)" href="<%= story.path %>" >
-        <div class="single-article-video-duration"><img src="<%= asset_path("video-camera.svg") %>" /><%= story.video_duration_in_minutes %></div>
-      </a>
-    <% end %>
-    <% organization = story.organization %>
-    <% if organization && !@organization_article_index %>
-      <div class="article-organization-headline">
-        <a href="/<%= organization.slug %>" class="article-organization-headline-inner"><img src="<%= organization.profile_image_90 %>"><%= organization.name %></a><a class="org-headline-filler" href="<%= story.path %>">&nbsp;</a></div>
-    <% end %>
-    <a href="<%= story.user.path%>" class="small-pic-link-wrapper">
-      <div class="small-pic" >
-        <% if story.cached_tag_list_array.include?("hiring") && story.organization_id.present? && !@organization_article_index %>
-          <img src="<%= ProfileImage.new(story.organization).get(90) %>" alt="<%= story.organization.username %> profile" />
-        <% else %>
-          <img src="<%= ProfileImage.new(story.user).get(90) %>" alt="<%= story.user.username %> profile" />
-        <% end %>
-      </div>
+<div class="single-article single-article-small-pic">
+  <% if story.video.present? && story.video_thumbnail_url.present? %>
+    <a class="single-article-video-preview" style="background-image:url(<%= story.cloudinary_video_url %>)" href="<%= story.path %>">
+      <div class="single-article-video-duration">
+        <img src="<%= asset_path("video-camera.svg") %>" /><%= story.video_duration_in_minutes %></div>
     </a>
-    <a href="<%= story.path %>" class="small-pic-link-wrapper index-article-link" data-preload-image="<%=cloud_cover_url(story.main_image) %>" id="article-link-<%= story.id%>">
-      <div class="content">
-        <h3>
-            <%= render "articles/tag_identifier", story: story, tag: @tag %>
-        </h3>
-      </div>
-    </a>
-    <h4>
-      <a href="<%= story.user.path %>"><%= story.user.name %>・<%= story.readable_publish_date %></a>
-      
-    </h4>
-    <div class="tags">
-      <% story.cached_tag_list_array.each do |tag| %>
-        <a href="/t/<%= tag %>"><span class="tag">#<%= tag %></span></a>
+  <% end %>
+  <% if story.organization && !@organization_article_index %>
+    <div class="article-organization-headline">
+      <a href="/<%= story.organization.slug %>" class="article-organization-headline-inner"><img src="<%= organization.profile_image_90 %>"><%= organization.name %>
```

Comment:
```suggestion
      <a href="/<%= story.organization.slug %>" class="article-organization-headline-inner"><img src="<%= organization.profile_image_90 %>"><%= story.organization.name %>
```

Reply:
Good catch, doesn't the part before that also have to be changed? Can you add that to your suggestion so I can commit it on Github.

Reply:
Oh nice catch! updated.